### PR TITLE
VTCCA-2976 file upload issue rework with extra validation

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

File: app/actions/AuthenticatedAction.scala [hunk 1]
```diff
@@ -71,31 +68,36 @@ class AuthenticatedAction @Inject()(override val messagesApi: MessagesApi,
                   body: BasicAuthenticatedRequest[A] => Future[Result])
                 (implicit request: Request[A], hc: HeaderCarrier): Future[Result] = {
     def handleError: PartialFunction[Throwable, Future[Result]] = {
-      case _: NoActiveSession =>
+      case _: InsufficientEnrolments    =>
```
Comment (darrell-plant) [inline app/actions/AuthenticatedAction.scala]: This has the issue we saw the other day where the body will be executed twice.

Comment (PeterPerhac) [reply app/actions/AuthenticatedAction.scala]: no , i saw that when i was working on auth library stuff, and i believe I did fix the double-running of the body, it's somewhere lower down in this file

**Comment by PeterPerhac** [comment]
that looks like the service is getting neater and slimmer by the minute! 👍 nice

**Comment by darrell-plant** [comment]
👍  after code walk-through

**Pull request merged**